MinVer/build/MinVer.props

```diff
@@ -0,0 +1,15 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects Condition="'$(MSBuildAssemblyVersion)' == '' Or '$(MSBuildAssemblyVersion)' &lt; '16.0'">$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <MinVerBuildMetadataENV>$(MinVerBuildMetadata)</MinVerBuildMetadataENV>
+    <MinVerMajorMinorENV>$(MinVerMajorMinor)</MinVerMajorMinorENV>
+    <MinVerTagPrefixENV>$(MinVerTagPrefix)</MinVerTagPrefixENV>
+    <MinVerVerbosityENV>$(MinVerVerbosity)</MinVerVerbosityENV>
+    <MinVerVersionOverrideENV>$(MinVerVersionOverride)</MinVerVersionOverrideENV>

```

Comment:
This is something I had thought of while working on this. If there was another package they built that was getting into MSBuild stuff, then I think there's an argument to consider that they'd need to also be a good MSBuild citizen and conditionally initialize things, much as I do in https://github.com/Particular/Particular.Packaging/blob/master/src/Particular.Packaging/build/Particular.Packaging.props.

If that was done, then the value from the env variable would still have the chance to override things.

Reply:
However, while thinking through this, there is one thing that doesn't quite work: Directory.Build.props files. I just doublechecked, and it looks like those get imported before any of the package props files do, so if a property was set in there, that would wipe out the env variable value before it could be captured. It's much less likely that you'd be doing "proper" conditional value setting in a file like that, so that does seem to be a bit of a problem for this more simplified approach.

Let me think on this a bit and see what I can come up with.